docu/report/aufbau.tex

```diff
@@ -24,12 +60,36 @@ \subsection{Antenne}
+Zur Synchronisation des Signals benutzt LTE sowohl ein Primary Synchronisation Signal (PSS) als auch ein Secondary Synchronisation Signal (SSS).~\cite[S.~180]{etsi2021136}
+
+\subsubsection{Framestrukturtyp}
+Der Framestrukturtyp 1, der für Vollduplex und Halbdublex FDD gilt, hat eine Länge von 10 ms und besteht aus 10 Subframes, die jeweils eine Länge von $1ms$ haben und von 0 bis 9 durchnummeriert sind. Bei Subframes, die die Frequenz $\Delta f=2.5kHz$, $\Delta f=7.5kHz$ oder $\Delta f=15kHz$ besitzen wird ein Subframe in zwei Slots unterteilt, wo die länge jedes Slots $0.5ms$ beträgt. Bei einer Subframefrequenz von $\Delta f=1.25kHz$ besteht der Subframe aus keinen Slots und seine Länge beträgt $1ms$. Wenn die Übertragungsfrequenz bei $\Delta f=0.37kHz$ liegt, besitzt ein Slot eine Länge von $3ms$. Eine Periode ist $40ms$ lang und besteht dem nach aus 12 Slots, die von 0 bis 13 durchnummeriert sind. Die Übertragung beginnt somit bei jedem vierten Frame. Außerdem stehen für FDD 10 Subframes, 20 Slots oder bis 60 Subslots zur Verfügung.
+
+\subsubsection{Primary Synchronisation Signal (PSS)}
+ PSS wird in jeden Frame zweimal übertragen, und zwar im ersten und 10 Slot. Innerhalb jedes Slots wird das PSS im letzten OFDM Symbol übertragen. Was UE damit erreicht mit dem PSS erreicht ist eine Subframe Synchronisation, eine Slot Synchronisation sowie eine Symbol Synchronisation. Außerdem kann die Mitte des jeweiligen Kanals bestimmt werden in der Frequenz Domain. Ebenfalls wird die passenden PCI aus den drei verschiedenen PCI erkannt.
+ Die Sequenz des PSS wird generiert durch die Zadoff-Chu erzeugt. Zadoff-Chu: $$ 	\operatorname{d_u}(n)=\begin{cases} e^{-j\frac{\pi un(n+1)}{63}}, & n=0,1,...,30 \\ e^{-j\frac{\pi u(n+1)(n+2)}{63}}, & n=31,32,...,61 \end{cases}$$~\cite[S.~181]{etsi2021136}. Bei einer MBMS-Zelle, die die Rahmenstruktur 1 verwendet, wird das Synchronsignal im Slot 0 in jedem vierten Subframe übertragen.
+
+\subsubsection{Secondary Synchronisation Signal (SSS)}
+Die für den sekundären Synchronisationssignal verwendete Sequenzen d(0),…,d(61) von zwei verknüpften Binärsequenzen der Länge 31. Die verknüpften Sequenzen werden mit einer Verschlüsselungssequenz aus dem ersten Synchronisationssignal verwurschtelt. Die Zuordnung der Sequenz zu den Resource-Elementen hängt von der Framestruktur ab. Bei einem Subframe des Framestrukturtyp 1 wird für das sekundäre Synchronisationssignal derselbe Antennenanschluss wie für das erstes verwendet. ~\cite[S.~183]{etsi2021136}
```

Comment:
Das `\cite` ist Teil des Satzes und kann auch als solches angesprochen werden. Daher sollte es zum einen _vor_ dem Satzpunkt gesetzt werden, zum anderen kannst du auch in deinem Satz darauf eingehen.
z.B.: "Wie in~\cite{Relativitätstheorie} wird eine Theorie zum Zusammenhang zwischen Raum und Zeit vorgestellt".

Btw. das `~` in LaTeX ist ein "non-breaking-space", d.h. LaTeX darf das "[12]" nicht auf eine neue Zeile setzen. Es ist somit nicht zielführend davor noch ein normales Leerzeichen zu setzen.
Beispiele: https://tex.stackexchange.com/a/74354

```suggestion
Die für den sekundären Synchronisationssignal verwendete Sequenzen d(0),…,d(61) von zwei verknüpften Binärsequenzen der Länge 31. Die verknüpften Sequenzen werden mit einer Verschlüsselungssequenz aus dem ersten Synchronisationssignal verwurschtelt. Die Zuordnung der Sequenz zu den Resource-Elementen hängt von der Framestruktur ab. Bei einem Subframe des Framestrukturtyp 1 wird für das sekundäre Synchronisationssignal derselbe Antennenanschluss wie für das erstes verwendet~\cite[S.~183]{etsi2021136}.
```